shared/markdown/generate-emoji-parser.js

```diff
@@ -54,6 +57,15 @@ function genEmojiData() {
     }
   })
 
+  // Add aliases after all default short names have been added. Otherwise, :man-woman-boy:’s
+  // :family: alias will take the place of the default :family: emoji, and they are not the same.
+  emojiData.forEach(emoji => {
+    emoji.short_names.forEach(name => addEmojiLiteral(emoji.unified, name))
```

Comment:
Hmm, is `short_name` part of `short_names`? If so, we may be needlessly bloating `emojiLiterals`, since it's an array and not a set...

Reply:
I don't think it'll hurt to have a dupe for each default short name in the regular expression that's used to build: https://github.com/keybase/client/blob/master/shared/markdown/generate-emoji-parser.js#L64

But I can exclude it if you think it's necessary.

Reply:
I went ahead and added this.